esp32_s3_lcd_ev_board/priv_include/bsp_sub_board.h

```diff
@@ -0,0 +1,56 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: CC0-1.0
+ */
+
+#pragma once
+
+#include "esp_err.h"
+#include "esp_lcd_types.h"
+
+#include "sdkconfig.h"
```

Comment:
sdkconfig.h is not needed in this file

Reply:
Fixed, PTAL [here](https://github.com/espressif/esp-bsp/pull/89/commits/abc7099df435b938ce2cd02f1ecafc7ca546c79c#diff-825e2d50928be7919de09e2fcbdf9f5d8967d785e1e7134d9e9bde04fc8ea5d6R9-R11).

Reply:
> @Lzw655 I am preparing SquareLine files. Please, are anywhere images of the both supported screens? I need it for SquareLine projects and I cannot find it.

Our colleague Zhang Wei has created a squareline project for this development board, how about asking him to raise a PR?

Reply:
> Our colleague Zhang Wei has created a squareline project for this development board, how about asking him to raise a PR?
@Lzw655 Oh, I didn't know, that somebody prepared it. It is a little hurry, because, there is waiting customer for it and the SquareLine relese will be soon. I made it yesterday here: https://github.com/espressif/esp-bsp/tree/SquareLine/add_esp32_s3_ev_board/SquareLine/boards but we can make PR on work of your colleague. It douesn't matter, which branch will be merged, the result should be same :-) I was only waiting for merge this BSP...